release-content/0.16/release-notes/_release-notes.toml

```diff
@@ -71,6 +71,13 @@ contributors = []
 prs = [16427]
 file_name = "16427_Use_multidraw_for_opaque_meshes_when_GPU_culling_is_in_use.md"
 
+[[release_notes]]
+title = "Occlusion culling"
+authors = ["@JMS55", "@pcwalton"]
```

Comment:
Listed in chronological order here.

Reply:
pcwalton should go first imo, I didn't work on any of the actual parts of this in 0.16.